### ISSUE TITLE
Error Message on Modal when Respose is not Sucessful

### ISSUE THREAD

**Issue by aayush6194** [LSBDCMonroe/Financial-App#4]
when res.sucess === false, modal should pop up and display the res.message

**Comment by ZachPoole** [comment]
UserStartPage was already done.
ClassDashboard updated.
Waiting on AddStudent route before adding Error message.

**Comment by ZachPoole** [comment]
Do you want the alert() or embedded into the UserModal component?
